demo/README.md

```diff
@@ -82,41 +83,64 @@ http_proxy_port: 900
+
+1. Ensure that the Device is initialized and onboarded as per the instructions listed in [Running Demo](#running-demo) section by enabling the 'Resale' protocol.
+
+2. Get the new Ownership Voucher using the Owner API `GET /api/v1/owner/newvoucher/?id=<device_guid>`.
+
+4. Start the FIDO IoT Reseller Sample as per the steps outlined in [Reseller README](reseller/README.md). Prior to the startup, configure the FIDO IoT Reseller to use the FIDO IoT Owner's keystore. This can be done by copying the \<fido-iot-src\>/demo/owner/owner_keystore.p12 into \<fido-iot-src\>/demo/reseller/ and renaming it to reseller_keystore.p12. Additionally,  update the property 'reseller_keystore_password' in \<fido-iot-src\>/demo/reseller/reseller.env, with value of 'owner_keystore_password' in \<fido-iot-src\>/demo/owner/owner.env.
+
+5. Add the new size 0 ownership voucher to the reseller database table `RT_DEVICES` using reseller API `POST /api/v1/resell/vouchers/<serial_number>`
+
+6. Add a new set of Owner PEM-formatted public keys in `RT_CUSTOMER` table using reseller API `POST /api/v1/resell/keys/?alias=<keystore_alias>`. Refer to [Generating Key-Pair](#generating-key-pair) for generating PEM-formatted key-pairs.
+
+7. Extend the voucher using the reseller API `GET /api/v1/resell/vouchers/<serial_number>?id=<customer_id>`.
+
+8. Start the FIDO IoT Owner Sample as per the steps outlined in [Owner README](owner/README.md). Prior to the startup, if an instance of FIDO IoT Owner is already running on the same port, terminate the previous instance. Configure the FIDO IoT Owner to use the new Owner's key-pairs. This can be done by deleting any existing 'alias' from the \<fido-iot-src\>/demo/owner/owner_keystore.p12, and inserting the key-pairs created in Step-6. Refer to [Inserting Keys into Keystore](#inserting-keys-into-keystore) and [Removing an Existing Key-Pair from Keystore](#removing-an-existing-key-pair-from-keystore). Additionally,  update the properties 'owner_keystore' and 'owner_keystore_password' in \<fido-iot-src\>/demo/owner/owner.env, with the keystore file-system path and password, respectively.
+
+9. Complete Transfer Ownership 1 and 2 (TO1 and TO2) by starting the FIDO IoT HTTP Java Device Sample.
 
 # Service info setup between FIDO IoT HTTP Java Device Sample and FIDO IoT Owner Sample
 
-The FIDO IoT HTTP Java Device Sample curretly supports `sdo_sys` module for interpreting received owner service info and `devmod` module to share device service info with Owner.
+The FIDO IoT HTTP Java Device Sample currently supports `sdo_sys` module for interpreting received owner service info and `devmod` module to share device service info with Owner.
```

Comment:
service info => ServiceInfo

(for all other place it might be getting used, as that the way it is referred to in the protocol spec)

Reply:
Fixed.